src/Application/Booking/Query/CheckBookingQuery.php

```diff
@@ -24,7 +26,7 @@ private function __construct(
     {
     }
 
-    public static function fromArray(UuidIdentifierInterface $uuid,\stdClass $data): self
+    public static function fromArray(UuidIdentifierInterface $uuid, stdClass $data): self
```

Comment:
it is better to use DTO here instead of stdClass , bcz we do not know stdClass of what ?